packages/client/src/events/call.ts

```diff
@@ -58,12 +58,16 @@ export const watchCallRejected = (call: Call) => {
         .every((m) => rejectedBy[m.user_id]);
       if (everyoneElseRejected) {
         call.logger('info', 'everyone rejected, leaving the call');
-        await call.leave({ reason: 'ring: everyone rejected' });
+        await call.leave({
+          reject: true,
+          reason: 'cancel',
+          message: 'ring: everyone rejected',
```

Comment:
This was working fine even before my fix (due to https://github.com/GetStream/stream-video-js/blob/main/packages/client/src/Call.ts#L583), but this makes it more explicit